typeDefs.js

```diff
@@ -104,6 +149,33 @@ const typeDefs = gql`
     getAllUsers: [User]
     getUserByToken: User
     getUserById(id: ID!): User
+
+    getAllProducts: [Products!]!
```

Comment:
Please, merge `getAllProducts` with `getProductsByOptions` and leave only one query `getProducts` that will take all required for filtering arguments

Reply:
Merged. Now It is one service getProducts